tools/contrib/fsurdat_modifier.py

```diff
@@ -0,0 +1,356 @@
+#! /usr/bin/env python
+"""
+|------------------------------------------------------------------|
+|---------------------  Instructions  -----------------------------|
+|------------------------------------------------------------------|
+
+Instructions for running on Cheyenne/Casper:
+
+load the following into your local environment
+    module load python
+    ncar_pylib
+
+-------------------------------------------------------------------
+To see available options:
+    ./fsurdat_modifier.py --help
+-------------------------------------------------------------------
+
+This script reads a surface dataset (fsurdat file) and outputs a
+modified copy of the same file.
+
```

Comment:
Maybe these notes can clarify how this is different than the `modify_singlept_site_neon.py` script that @negin513 created and what it should be used for? I have two other high level questions:

1. Is there code that should be shared between these scripts (to @billsacks' point about putting this in the python directory.
2. Should we consider renaming the wrapper scripts we're calling so they are descriptive and consistent?

Reply:
@wwieder thanks. I have added clarifications that will appear in my next commit.

1. Ideally, yes, and we will continue to keep that in mind during development.
2. I'm not attached to existing names.